### PR TITLE
Docs: Incorporate changes from 'go' SDK, add accents in tutorial

### PR DESCRIPTION
**Pull request by akcano**

### CODE REVIEW

File: docs/explanation/essentials/moving-projects.rst [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+       Project                 Workshop  Status  Notes
+       /home/user/project      golang    Ready   -
+       /home/user/PROJECT      golang    Ready   -
+
+
+Now, these are two independent workshops that happen to have the same name,
+not a single workshop that is somehow shared by multiple project directories.
+
+Again, it's up to you to update the new project's metadata
```
Comment (dmitry-lyfar) [inline docs/explanation/essentials/moving-projects.rst]: I think this (and a similar comment about the project's files and settings) is unnecessary to clarify? I suppose this comment and the one above use different terms to refer the same notion (project's files and settings vs. project's metadata)?

File: docs/how-to/git-workshop.rst [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+   $ cd project
+
+
+.. code-block:: yaml
+   :caption: .workshop.golang.yaml
+
+   name: golang
+   base: ubuntu@22.04
+   sdks:
+   go:
```
Comment (dmitry-lyfar) [inline docs/how-to/git-workshop.rst]: `go:` no indent?

File: docs/how-to/git-workshop.rst [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+amend the workshop definition appropriately:
+
+.. code-block:: yaml
+   :caption: .workshop.golang.yaml
+   :emphasize-lines: 5
+
+   name: golang
+   base: ubuntu@22.04
+   sdks:
+   go:
```
Comment (dmitry-lyfar) [inline docs/how-to/git-workshop.rst]: `go:` no indent?

File: docs/how-to/git-workshop.rst [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+Suppose you want to fix an issue in a separate branch,
+using newer features from the same SDK:
+
+.. code:: console
+
+   $ git worktree add ../hotfix
+   $ cd ../hotfix
+
+
+To update the SDK version,
```
Comment (dmitry-lyfar) [inline docs/how-to/git-workshop.rst]: To discuss: we should not change an SDK version in this how-to as the howto's point is to show a full (major) cycle with a worktree/workshop without disturbing main environment: 

1. Create a worktree or checkout another branch for a quick fix or test (no changes to the main work branch).
2. Launch a workshop in the new worktree.
3. Move a worktree with `git worktree move` and see how workshop follows nicely.
4. Discard the workshop and remove the worktree when the job is done; come back to your primary branch.

File: docs/explanation/essentials/moving-projects.rst (hunk 1 above)
Comment (akcano) [reply docs/explanation/essentials/moving-projects.rst]: Stating the obvious is OK for an explanation, as it usually needs to make a [couple of steps back](https://www.lesswrong.com/posts/AcbtSuGYB4jWGYk6x/some-notes-on-solving-hard-problems#Step_back_regularly) and occasionally reiterate the basics.

However, the discrepancy in terms can and will be addressed.

**Review by dmitry-lyfar — APPROVED** [review]

**Pull request merged**